src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1168,9 +1168,17 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                                 while (($remainingMessage.Length + $prefixLength) -gt $windowWidth) {
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-linux`
Tests for $ErrorView.ConciseView tests.Long exception message gets rendered
```powershell
Expected like wildcard '*1234567890*' to match <empty>, but it did not match.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Formatting/ErrorView.Tests.ps1: line 100
100:             $e | Should -BeLike "*$msg*"
```